### DISCUSSION TITLE
Meshing error module -M

### DISCUSSION THREAD

**Discussion by ap264** [neperfepx/neper#543] [Q&A] [answered]
I have created the tessalation and mesh using the neper software and exported the mesh to abaqus using inp format. After performing simulation, abaqus gives me an error saying "some elements are either negitive, zero or small". I would like to recity this mistake and will be deeply appreciated if some assistance is provided. The following commands are used to generate the tessalation and mesh

neper -T - n -70 -o mytess
neper -M -elttype tet -rcl 0.6 -dim 2 -format inp
![neper](https://user-images.githubusercontent.com/109927295/186261960-4fd2667f-328e-408f-8daf-10a96b25c6a3.PNG)

**Comment by rquey** [comment, marked as answer]
@ap264, as far as possible, please copy-paste text as text, not images (they don't look good, prevent us from selecting text (when necessary) and take unnecessary space).

I have no real clue about this. Some elements may have a small area, but none have a zero or a negative area... Maybe Abaqus is only printing this message because of the small elements, and this is not causing further problems?
(related: https://github.com/neperfepx/neper/discussions/477#discussioncomment-3472467)
